Examples/Framework/include/ActsExamples/EventData/SimSeed.hpp

```diff
@@ -0,0 +1,33 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Seeding/Seed.hpp"
+#include "ActsExamples/EventData/SimSpacePoint.hpp"
+
+#include <map>
+#include <vector>
+
+namespace ActsExamples {
+/// Container of sim seed
+using SimSeedContainer = std::vector<std::vector<Acts::Seed<SimSpacePoint>>>;
+
+/// Struct for the grouped seed index
+struct GroupedSeedIdx {
```

Comment:
```suggestion
struct GroupedSeedIndex {
```
I see no need to abbreviate here. It also matches better with the existing `Index` type name.

Reply:
Fixed.